packages/web/src/js/constants.ts

```diff
@@ -4,4 +4,5 @@ export const ARIA_CONTROLS_ATTRIBUTE = 'aria-controls';
 export const NAME_DATA_TARGET = 'data-target';
 
 export const CLASSNAME_EXPANDED = 'is-expanded';
-export const CLASSNAME_COLLAPSED = 'is-collapsed';
+export const CLASSNAME_OPEN = 'is-open';
+export const CLASSNAME_TRANSITION = 'is-transition';
```

Comment:
Why is here this class? We are not using it anywhere in SCSS.

Reply:
This class is here for prevents 'blink' content when expanding and collapsing. Class is set only for expanding and collapsing.